kong/cmd/utils/prefix_handler.lua

```diff
@@ -248,6 +248,18 @@ local function compile_conf(kong_config, conf_template, template_env_inject)
     }
   }
 
+  local kong_proxy_access_log = kong_config.proxy_access_log
+  if kong_proxy_access_log ~= "off" then
+    compile_env.proxy_access_log_enabled = true
+  end
+  if kong_proxy_access_log then
+    -- example: 'logs/some-file.log apigw_json'
+    local _, custom_format_name = string.match(kong_proxy_access_log, "^(%S+)%s(%S+)")
+    if custom_format_name then
+      compile_env.proxy_access_log_custom = true
```

Comment:
```suggestion
      compile_env.custom_proxy_access_log = true
```

Reply:
^ just careful to update this everywhere if you commit the change :)

Reply:
Done